modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -162,7 +162,7 @@ The sequential access method assumes that:
 . There is a direct correlation between the aggregated records and the items in the list.
 
 You can also access random records by specifying the iteration number of the foreach, saving you the need to iterate through all records. +
-The foreach scope exposes a `records` variable. This variable is an immutable list used by foreach to keep track of the iteration and provides a random access list that is accessible across the batch aggregator.
+The foreach scope exposes a `records` variable. This variable is an immutable list used by foreach to keep track of the iteration and provides a random access list that is accessible across the Batch Aggregator.
```

Comment:
there used to be an example for this. Also this doesn't make it clear to know when to use the `record` variable inside a foreach or when to use the `records` variable for random access.